File: main/src/main/java/net/fwitz/math/main/binary/split/functions/exponential/SplitExp1OverZ.java
package net.fwitz.math.main.binary.split.functions.exponential;

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitExp1OverZ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("exp(1/z)")
                .computeFn(z -> z.inverse().exp())
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/exponential/SplitExp1OverZ.kt
package net.fwitz.math.main.binary.split.functions.exponential

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitExp1OverZ {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("exp(1/z)")
        .computeFn { z -> z.inverse.exp }
        .render()
}